### PR TITLE
Feature/pub 380 update docusaurus version

### PR DESCRIPTION
**Pull request by Skretzschmann**

### CODE REVIEW

File: website/docs/containerization/05-overview.md [hunk 1]
```diff
@@ -4,13 +4,14 @@ title: Container Overview
...
 
-<VideoContainer
-  list={[{
-   src: "https://www.youtube-nocookie.com/embed/jq9wLQpDtvQ",
-   title: "Container Overview"
-  }]}
+<video
+  url={[
+    "https://www.youtube-nocookie.com/embed/jq9wLQpDtvQ",
+  ]}
```
Comment (tmeier-a9s) [inline website/docs/containerization/05-overview.md:10]: Not sure about this, the look and feel is different (e.g. no youtube player controls available)
What was the issue with `VideoContainer`?

File: website/package.json [hunk 2]
```diff
@@ -7,14 +7,15 @@
     "swizzleMDX": "docusaurus swizzle @docusaurus/theme-classic MDXComponents --danger"
   },
   "dependencies": {
-    "@anynines/a9s-design-system": "^0.2.2",
-    "@docusaurus/core": "^2.0.0-beta.6",
-    "@docusaurus/preset-classic": "^2.0.0-beta.6",
+    "@anynines/a9s-design-system": "*",
+    "@docusaurus/core": "*",
+    "@docusaurus/preset-classic": "*",
```
Comment (tmeier-a9s) [inline website/package.json:12]: use explicit versions

**Review by tmeier-a9s — CHANGES_REQUESTED** [review]

File: website/docs/10-overview.md [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
 ## Related Videos
 <VideoContainer
-list={[
-{src: "https://www.youtube-nocookie.com/embed/H5OY7IDjPZU", title: "Overview"},
-]}></VideoContainer>
-
+    list={[{
+      src: "https://www.youtube-nocookie.com/embed/H5OY7IDjPZU", 
+      title: "Overview"
+  }]}
```
Comment (tmeier-a9s) [inline website/docs/10-overview.md:12]: please align closing brackets with the right indentation. they should match `list`. also you are mixing 4 spaces with 2 spaces indentations. keep using 2 as everywhere else.

File: website/docs/containerization/05-overview.md [hunk 4]
```diff
@@ -6,11 +6,12 @@ title: Container Overview
 
 <VideoContainer
   list={[{
-   src: "https://www.youtube-nocookie.com/embed/jq9wLQpDtvQ",
-   title: "Container Overview"
+    src: "https://www.youtube-nocookie.com/embed/jq9wLQpDtvQ",
+    title: "Container Overview"
```
Comment (tmeier-a9s) [inline website/docs/containerization/05-overview.md:10]: wrong indentation here too. now you are using 1 instead of 2.

File: website/src/theme/MDXComponents/A.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (tmeier-a9s) [inline website/src/theme/MDXComponents/A.js:1]: please set up `prettier` and `eslint` locally and run them over your files. Those are not properly formatted files.

File: website/src/theme/MDXComponents/Video.js [hunk 6]
```diff
@@ -44,3 +44,5 @@ export default function Video({ src, title, size, faded }) {
     </Grid>
   )
 }
+
+Video.isMDXComponent = true
```
Comment (tmeier-a9s) [inline website/src/theme/MDXComponents/Video.js:48]: was this addition needed?

File: website/src/theme/MDXComponents/VideoContainer.js [hunk 7]
```diff
@@ -36,9 +36,8 @@ export default function VideoContainer({ title, list }) {
       <Grid container spacing={4}>
         {videos}
       </Grid>
-      <Divider style={{ paddingTop: '1.8em' }} variant='fullWidth'></Divider>
     </div>
   )
 }
 
-VideoContainer.isMDXComponent = true
+VideoContainer.isMDXComponent = true
```
Comment (tmeier-a9s) [inline website/src/theme/MDXComponents/VideoContainer.js:43]: same here, was it needed?

File: website/src/theme/MDXComponents/index.js [hunk 8]
```diff
@@ -1,85 +1,32 @@
...
-
-  return element
-}
+import React from 'react';
+import MDXHead from '@theme/MDXComponents/Head';
+import MDXCode from '@theme/MDXComponents/Code';
+import MDXA from '@theme/MDXComponents/A';
+import MDXPre from '@theme/MDXComponents/Pre';
+import MDXDetails from '@theme/MDXComponents/Details';
+import MDXHeading from '@theme/MDXComponents/Heading';
```
Comment (tmeier-a9s) [inline website/src/theme/MDXComponents/index.js:7]: whats the difference between `Heading` and `MDXHeading`? `Heading` still used? if not, can it then be deleted? same for the other files

File: yarn.lock [hunk 9]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (tmeier-a9s) [inline yarn.lock:1]: remove this file

File: website/static/index.html [hunk 10]
```diff
@@ -1,18 +0,0 @@
-<!DOCTYPE html>
```
Comment (tmeier-a9s) [inline website/static/index.html:1]: why is this file deleted? whats with the rest of `static` folder?

**Review by tmeier-a9s — CHANGES_REQUESTED** [review]

File: website/static/index.html (hunk 10 above)
Comment (Skretzschmann) [reply website/static/index.html:1]: The static page caused errors after the update / the redirect did not work.

Has been replaced by adding Slug in website/docs/10-overview.md 
and making a redirect in
website/docusaurus.config.js

Not sure about the rest of the files in the Static folder tbh.

File: website/src/theme/MDXComponents/VideoContainer.js (hunk 7 above)
Comment (Skretzschmann) [reply website/src/theme/MDXComponents/VideoContainer.js:43]: The line is inlcuded in the current development branch, but I just checked project still builds and looks normal after the removal.

File: website/src/theme/MDXComponents/A.js (hunk 5 above)
Comment (Skretzschmann) [reply website/src/theme/MDXComponents/A.js:1]: ?? are you referring to the .js files or .md files?

the .js files in MDXComponents have been auto generated with exception being index, video and videocontainer

Comment (tmeier-a9s) [reply website/src/theme/MDXComponents/A.js:1]: how are they autogenerated? by what?
even so, they should then be reformatted by a linter to match the coding standard.

**Review by tmeier-a9s — CHANGES_REQUESTED** [review]

**Review by tmeier-a9s — APPROVED** [review]
